data_pipeline/crossref_event_data/etl_crossref_event_data_util.py

```diff
@@ -220,6 +224,13 @@ def per_doi_download_page_etl(
         schema: list,
         journal_previous_timestamp: datetime,
 ):
+    http.client.HTTPConnection.debuglevel = 1
```

Comment:
Is this to get the output in the error case?

Enabling debug logging at the lower level might log things we may not want to log (e.g. credentials).
(It also affects not just this DAG I think)

I would suggest to just get it from the response (I guess inside `get_crossref_data_single_page`).
We should have the full response with the status code available.

Reply:
OK I am checking thank you!